test/parser.test.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs');
+const path = require('path');
+const { parse } = require('csv');
+const {output, setWatched,setOutput,setProcessed} = require("../src/parser");
+const watch  = require("../src/watcher");
+const { processChange } = require('../src/parser');
+
+console.info("attempting wrong file test");
+test('checks a non cs file',()=>{
+    expect(processChange('../test_data/patient_data_5.json')).toBe(false);}
```

Comment:
Passes. Good.

The test doesn't test much, _but_ you are specific with what is being tested and you're using semantically correct assertions. Good job.